src/config/acbs.config.ts

```diff
@@ -0,0 +1,36 @@
+import { registerAs } from '@nestjs/config';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+export interface AcbsConfig {
```

Comment:
This interface isn't imported anywhere, but if I don't include `export` then the build fails with error
```
Default export of the module has or is using private name 'AcbsConfig'
```